dp-search-reindex-tracker.nomad

```diff
@@ -0,0 +1,140 @@
+job "dp-search-reindex-tracker" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "service"
+
+  update {
+    stagger          = "60s"
+    min_healthy_time = "30s"
+    healthy_deadline = "2m"
+    max_parallel     = 1
+    auto_revert      = true
+  }
+
+  group "web" {
```

Comment:
Do we need the web group?

It only runs in publishing

Reply:
Awesome spot. Done